### PR TITLE
Created Telegram Bot

### PR DESCRIPTION
**Pull request by yankovskiid**

### CODE REVIEW

**Comment by yankovskiid** [comment]
@anatoliliotych please review code

File: Telegram_bot-master/bot.rb [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+require_relative 'token.rb'
+
+def get_current_weather(city)
+  options = { units: 'metric', APPID: OPEN_WEATHER_TOKEN }
+  req = OpenWeather::Current.city(city, options)
+  "description:#{req['weather'].first['description']}, tempreture:#{req['main']['temp_min']}"
+end
+
+flag = 0
+binding.pry
```
Comment (anatoliliotych) [inline Telegram_bot-master/bot.rb]: it shouldn't be here

File: Telegram_bot-master/bot.rb [hunk 2]
```diff
@@ -0,0 +1,38 @@
+require 'rubygems'
+require 'telegram/bot'
+require 'logger'
+require 'pry'
+
+require_relative 'commands.rb'
+require_relative 'token.rb'
```
Comment (anatoliliotych) [inline Telegram_bot-master/bot.rb]: use .env file with dotenv gem for storing token

File: Telegram_bot-master/bot.rb [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+
+def get_current_weather(city)
+  options = { units: 'metric', APPID: OPEN_WEATHER_TOKEN }
+  req = OpenWeather::Current.city(city, options)
```
Comment (anatoliliotych) [inline Telegram_bot-master/bot.rb]: what if this request fail? you should think about such things working with third party services!

File: Telegram_bot-master/commands.rb [hunk 4]
```diff
@@ -0,0 +1,5 @@
+
```
Comment (anatoliliotych) [inline Telegram_bot-master/commands.rb:1]: empty line should be removed!

File: Telegram_bot-master/commands.rb [hunk 5]
```diff
@@ -0,0 +1,5 @@
...
+HELP_COMMANDS = " /help - commands list \n /weather_current - waether forecast \n /mem - memos /n".freeze
```
Comment (anatoliliotych) [inline Telegram_bot-master/commands.rb:2]: you can store strings in yml file

**Review by anatoliliotych — CHANGES_REQUESTED** [review]

**Comment by yankovskiid** [comment]
@anatoliliotych please review code again

**Pull request merged**